src/Raven.Server/Web/System/AdminServerWideBackupHandler.cs

```diff
@@ -130,6 +131,9 @@ public Task GetServerWideBackupConfigurationCommand()
                 {
                     var backup = JsonDeserializationServer.ServerWideBackupConfiguration(backupBlittable);
                     backup.BackupDestinations = backup.GetDestinations();
+                    backup.IsEncrypted = backup.BackupEncryptionSettings != null &&
```

Comment:
Is this valid? It differs from the `IsBackupEncrypted` logic

Reply:
We only know that the configuration is encrypted